lib/Twig/NodeVisitor/Optimizer.php

```diff
@@ -25,6 +25,8 @@
     const OPTIMIZE_NONE = 0;
     const OPTIMIZE_FOR = 2;
     const OPTIMIZE_RAW_FILTER = 4;
+    const OPTIMIZE_CONCAT_CONST = 5;
```

Comment:
5 is not good. Binary flags must use powers of 2 for the value of each flag. So it should use `16`